portal/src/main/webapp/js/lib/SBGNViz.js/sample-app/index.html

```diff
@@ -75,11 +74,10 @@
                                 <b class="caret"></b>
                             </a>
                             <ul class="dropdown-menu">
-                                <li><a href="#" id="select-edit">Select/Edit</a></li>
+                                <!-- <li><a href="#" id="select-edit">Select/Edit</a></li>

```

Comment:
Is there any reason to keep the code? If not, could you just remove them instead of commenting?